src/docs/content/observability/observability-example/setup.md

```diff
@@ -0,0 +1,236 @@
+---
+title: "Setup"
+date: 2020-03-10T13:29:23-06:00
+draft: false
+weight: 3
+tags:
+  - OCI Logging
+  - OCI Notifications
+  - OCI Service Connector Hub
+  - Deploy MuShop
+  - fluentd
+  - logging
+  - alarms
+  - notifications
+---
+
+### Introduction
+
+In this section we will deploy MuShop, Oracle Cloud Infrastructure (OCI) Notifications, Logging and Service Connector Hub.
+
+### Deploy MuShop
+
+{{% alert style="information" icon="warning" %}}
+We will be deploying MuShop on Oracle Cloud Infrastructure Container Engine for Kubernetes (OKE)
+{{% /alert %}}
+
+- Create kubernetes namespace
+```bash
+kubectl create namespace mushop
+```
+- Deploy
+```
+cd deploy/complete/kubernetes
+```
+```text
+kubectl -n mushop apply -f mushop.yaml
+```
+- Verify if all the pods are in running status 
+```text
+kubectl -n mushop get pods
+```
+
+- Expose
+
+This command exposes the edge service on `localhost`.
+
+```text
+kubectl -n mushop port-forward svc/edge 8000:80
+```
```

Comment:
While this is definitely an option with a local `kubectl`, it presents some problems in the case of Cloud Shell users. Ideally each of the examples will function and be accessible via Cloud Shell. As an example of the two different options, you may consider the wording [here](https://oracle-quickstart.github.io/oci-cloudnative/quickstart/kubernetes/#deploy-mushop). Or perhaps even better would be the use of the [switcher](https://oracle-quickstart.github.io/oci-cloudnative/usage/#switcher) shortcode. Note that the ingress controller IP requires the [setup](https://oracle-quickstart.github.io/oci-cloudnative/quickstart/kubernetes/#setup) to be completed